ansible_runner/interface.py

```diff
@@ -83,7 +83,7 @@ def init_runner(**kwargs):
         if logfile:
             output.set_logfile(logfile)
 
-    if kwargs.get("process_isolation", False):
+    if kwargs.get("process_isolation", False) and kwargs.get("streamer") not in ('transmit', 'process'):
```

Comment:
I have this in my PR also: https://github.com/ansible/ansible-runner/pull/1079/files#diff-179ccecedba1c6ccf73b8c02f0321b252c018226e41ce82887a93ace762d2fbbR55

This line causes an astonishing amount of problems when interfacing with Runner from the CLI